src/_P033_Dummy.ino

```diff
@@ -61,6 +62,12 @@ boolean Plugin_033(uint8_t function, struct EventStruct *event, String& string)
       break;
     }
 
+    case PLUGIN_SET_DEFAULTS:
+    {
+      Settings.TaskDeviceTimer[event->TaskIndex] = Settings.Delay; // 'Old' default value (before TimerOptional was enabled)
```

Comment:
PLUGIN_SET_DEFAULTS is only called when adding a new instance.
When converting existing settings, you should increment the build nr and add a settings transition (in ESPEasyStorage I think)

Reply:
Originally, for all tasks using Interval, the setting gets the value from Settings.Delay if Interval is _not_ optional, and the value is set to 0. That is already applied to existing instances of Dummy device, but to mimic that behavior for new tasks, the value is now set when adding the task. The user can decide to set it to 0.
No reason to update the build nr, I'd think.

Reply:
We could opt for _not_ setting a default Interval for new Dummy device tasks, as in my normal use-case, that flush of the dummy values in the Info log is mostly annoying 😛

Reply:
I'm OK with not transitioning, but then you know there will be reported issues where there is a user that was relying on the dummy task running at some regular interval. :)